foyer/forcefield.py

```diff
@@ -721,14 +723,16 @@ def _write_references_to_file(self, atom_types, references_file):
                 warnings.warn("Reference not found for atom type '{}'."
                               "".format(atype))
         unique_references = collections.defaultdict(list)
-        for key, value in atomtype_references.items():
-            unique_references[value].append(key)
+        for key, values in atomtype_references.items():
```

Comment:
Could we update these to be `for atomtype, dois in ...`?

Reply:
👍